### PR TITLE
Limitations with mixed PGD text.

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

First pass - expand the limitations section with a specific prohibition.

### CODE REVIEW

File: product_docs/docs/pgd/5/limitations.mdx [hunk 1]
```diff
@@ -74,6 +74,12 @@ within commit scope rules.
 Consult the [Durability limitations](durability/limitations) section for a full
 and current listing.
 
+## Mixed PGD versions
+
+PGD has been developed to [enable rolling upgrades of PGD](upgrades) by allowing mixed versions of PGD to run during the upgrade process. 
+We expect users to only run mixed versions during upgrades and, once an upgrade has been started, that they complete that upgrade.
+We do not support mixed versions of PGD where there is no upgrade taking place.
```
Comment (LenzGr) [inline product_docs/docs/pgd/5/limitations.mdx:81]: Maybe a tad bit crisper:
```suggestion
We do not support running mixed versions of PGD where no upgrade occurs.
```

**Review by LenzGr — APPROVED** [review]
LGTM, thank you.

**Review by mwanner — DISMISSED** [review]
Sounds perfect to me, thanks!

File: product_docs/docs/pgd/5/limitations.mdx (hunk 1 above)
Comment (djw-m) [reply product_docs/docs/pgd/5/limitations.mdx:81]:
```suggestion
We do not support mixed versions of PGD when there is no upgrade taking place.
```

Should really be temporal.

Comment (djw-m) [reply product_docs/docs/pgd/5/limitations.mdx:81]:
```suggestion
We do not support running mixed versions of PGD other than while there is an upgrade taking place.
```

File: product_docs/docs/pgd/5/limitations.mdx [hunk 2]
```diff
@@ -74,6 +74,12 @@ within commit scope rules.
 Consult the [Durability limitations](durability/limitations) section for a full
 and current listing.
 
+## Mixed PGD versions
+
+PGD has been developed to [enable rolling upgrades of PGD](upgrades) by allowing mixed versions of PGD to run during the upgrade process.
```
Comment (djw-m) [inline product_docs/docs/pgd/5/limitations.mdx:79]:
```suggestion
PGD has been developed to [enable rolling upgrades of PGD](upgrades) by allowing mixed versions of PGD to operate during the upgrade process. 
```

**Pull request merged**